toolchain/sem_ir/formatter.cpp

```diff
@@ -500,7 +505,7 @@ auto Formatter::FormatSpecificRegion(const Generic& generic,
 auto Formatter::FormatSpecific(SpecificId id) -> void {
   const auto& specific = sem_ir_->specifics().Get(id);
   const auto& generic = sem_ir_->generics().Get(specific.generic_id);
-  if (!should_format_entity_(generic.decl_id)) {
```

Comment:
Is there a comment we could leave to explain why this doesn't call ShouldFormatEntity?

Reply:
I don't know why this didn't call `ShouldFormatEntity`, but the new behavior is equivalent.